### PR TITLE
5.3 policy kovalets

### PR DESCRIPTION
**Pull request by AndriyKovalets**

### CODE REVIEW

File: Provis.Core/Roles/WorkSpaceRoles.cs [hunk 1]
```diff
@@ -2,9 +2,9 @@
 {
     public class WorkSpaceRoles
```
Comment (YevheniiVovk) [inline Provis.Core/Roles/WorkSpaceRoles.cs]: Let's use enum for this

Comment (AndriyKovalets) [reply Provis.Core/Roles/WorkSpaceRoles.cs]: I've changed from const values to enum.

File: Provis.Core/Services/WorkspaceService.cs [hunk 2]
```diff
@@ -97,7 +97,7 @@ public async Task SendInviteAsync(InviteUserDTO inviteDTO, string ownerId)
             else
             {
 
-                if (checkRole.RoleId == WorkSpaceRoles.MemberId || checkRole.RoleId == WorkSpaceRoles.ViewerId)
+                if (checkRole.RoleId == (int)WorkSpaceRoles.MemberId || checkRole.RoleId == (int)WorkSpaceRoles.ViewerId)
```
Comment (YevheniiVovk) [inline Provis.Core/Services/WorkspaceService.cs]: Let's make check by convert to enum

File: Provis.WebApi/Controllers/WorkspaceController.cs [hunk 3]
```diff
@@ -51,6 +53,7 @@ public async Task<IActionResult> AcceptInviteUserAsync(int id)
 
         [Authorize]
         [HttpPost]
+        [WorkspaceRoles(new int[] { (int)WorkSpaceRoles.OwnerId, (int)WorkSpaceRoles.ManagerId })]
```
Comment (YevheniiVovk) [inline Provis.WebApi/Controllers/WorkspaceController.cs]: Let's set enum values as attribute values

Comment (AndriyKovalets) [reply Provis.WebApi/Controllers/WorkspaceController.cs]: I've changed it

File: Provis.Core/Services/WorkspaceService.cs (hunk 2 above)
Comment (AndriyKovalets) [reply Provis.Core/Services/WorkspaceService.cs]: this check was a temporary solution to check if the user is owner or maneger, now for it is policy, so i removed this check

**Review by Sergey988 — APPROVED** [review]

**Review by FeloniukM — CHANGES_REQUESTED** [review]
need bag fix

**Review by FeloniukM — APPROVED** [review]

**Pull request merged**